spec/controllers/queue_items_controller_spec.rb

```diff
@@ -159,4 +159,128 @@
     end
   end
 
+  describe 'PUT :update' do
+
+    let(:user) { user = Fabricate(:user) }
+    let(:queue_items) { queue_items = user.queue_items }
+    let(:first) { first = queue_items.first }
+    let(:sec) { sec = queue_items[1] }
+    let(:third) { third = queue_items.last }
+    before do
+      3.times do |i|
+        Fabricate(:queue_item, user: user, order_value: i+1)
```

Comment:
Also, You can use advantage of Fabrication sequences - [check this out](http://www.fabricationgem.org/#!sequences) - at the bottom especially. And then you could just do `Fabricate.times(3, :queue_item, user: user)`. I think it should work, haven't tried it yet.